includes/functions-html.php

```diff
@@ -359,8 +365,8 @@ function yourls_html_tfooter( $params = array() ) {
  * @param boolean $display false (default) to return, true to echo
  * @return string HTML content of the select element
  */
-function yourls_html_select( $name, $options, $selected = '', $display = false ) {
-	$html = "<select name='$name' id='$name' size='1'>\n";
+function yourls_html_select( $label, $name, $options, $selected = '', $display = false ) {
```

Comment:
I would not add the label as the first argument. Also, it may be good to add default empty value for compatibility.